cmd/root.go

```diff
@@ -23,6 +24,8 @@ type RootOptions struct {
 	NetNs                 string
 	UseWaitFlag           bool
 	TimeoutCloseWaitSecs  int
+	LogFormat             string
```

Comment:
Could we use the `ConfigureAndParse` function from "linkerd/linkerd2/pkg/flags" to configure the log level and format?

Reply:
@adleong I actually wanted to do that at first, however (probably due to my lack of profficiency with go), I didn't know how to properly use the existing cobra command structure at root.go and ConfigureAndParse structure itself, without having to change `main.go` and the command structure itself.
 I'll be working on it next, but if you guys have any examples you can share, I would appreciate it :smile:

Reply:
Upon closer inspection, you're right that this isn't an easy fit.  `ConfigureAndParse` looks like it is mostly intended for processes which don't do their own flag parsing (like the control plane components).  Rather than try to jam it in here, I agree with keeping what you already have.